include/cutlass/conv/kernel/default_conv2d_fprop.h

```diff
@@ -597,7 +623,7 @@ struct DefaultConv2dFprop <
     arch::CacheOperation::Always,
     IteratorB,
     SmemIteratorB,
-    arch::CacheOperation::Global,
+    arch::CacheOperation::Always,
```

Comment:
i don't think we need to change this.

Reply:
I change here because there is an assert in cp.async.cg says it only support 16B access size

Reply:
I see.

You need something like this for `B`: https://github.com/NVIDIA/cutlass/blob/master/include/cutlass/gemm/threadblock/default_mma.h#L427-L430

When it is 128bit use `Global`, use `Always` when it is smaller than 128bit.

Reply:
For unit tests, do I just need to add one more TestAllConv2d in fp16_sm75/80.cu and tf32_sm80.cu with one or several unalignment case, or should I add additional tests in all relevant cu files.

Reply:
just adding align=1,2,4 tests to  fp16_sm75/80.cu and align=1,2 to `tf32_sm80.cu`

Reply:
I add some unit tests in the new patch

Reply:
The newly added unit tests look good.  Have you tried running all the conv unit tests including the one you added?